templates/experiments/experiment_form.html

```diff
@@ -122,6 +122,17 @@ <h3 class="text-md font-bold">{% translate "Please note" %}</h3>
       </div>
     </div>
 {% endblock form %}
+{% block form_actions %}
+  <input type="submit" class="pg-button-primary mt-2" value="{{ button_text }}">
+  {% flag "experiment_versions" %}
+    <a href="{% url 'experiments:create_version' team.slug experiment.id %}" class="btn btn-secondary" style="float: right;">
```

Comment:
I don't think this is going to save the experiment - you'll loose changes.

You can use a normal form submit button and add a name attribute so that you tell which button got pressed:

```
<button type="submit" name="action" value="save_and_version">Create Version</button>
```

On the server side check `request.POST.get("action") == "save_and_version"`